examples/worldview/src/features/map/MapContainer.js

```diff
@@ -48,7 +48,7 @@ export const MapContainer = ({previewSize, deckProps, staticMapProps, debug, vie
       resolution={resolution}
       updateTimeCursor={timeMs => dispatch(updateTimeCursor(timeMs))}
       debug={debug}
-      viewportMinAxis={viewportMinAxis}
+      viewportMinAxis={Math.min(resolution.width, resolution.height)}
```

Comment:
Can we make this so that if the viewportMinAxis prop is provided it uses that, otherwise does this? There are situations where either is used.